.golangci.yml

```diff
@@ -13,8 +17,42 @@ linters-settings:
     enable=fieldalignment: true
   revive:
     rules:
-    - name: if-return
-      disabled: true
+      # The following rules are recommended https://github.com/mgechev/revive#recommended-configuration
+      - name: blank-imports
+      - name: context-as-argument
+      - name: context-keys-type
+      - name: dot-imports
+      - name: error-return
+      - name: error-strings
+      - name: error-naming
+      - name: exported
+        disabled: true  # TODO: Investigate if it should be enabled. Disabled for now due to many findings.
+      - name: if-return
+        disabled: true  # TODO: Investigate if it should be enabled. Disabled for now due to many findings.
+      - name: increment-decrement
+      - name: var-naming
+        disabled: true   # TODO: Investigate if it should be enabled. Disabled for now due to many findings.
+      - name: var-declaration
+      - name: package-comments
+        disabled: true  # TODO: Investigate if it should be enabled. Disabled for now due to many findings.
+      - name: range
+      - name: receiver-naming
+      - name: time-naming
+      - name: unexported-return
+      - name: indent-error-flow
+      - name: errorf
+      - name: empty-block
+        disabled: true  # TODO: Investigate if it should be enabled. Disabled for now due to many findings.
```

Comment:
If that is the recommendation we might want in a follow up fix the issues and then enable right?

Reply:
Yes, I think that would be good to do. You might want them. You might not. But I disabled the checks that found many / non trivial findings to keep the complexity and size of this pr smaller.